i18n/create_messages.py

```diff
@@ -34,15 +34,15 @@ def string_is_ascii(s):
     return True
   except UnicodeEncodeError:
     return False
-  
+
 def load_constants(filename):
   """Read in constants file, which must be output in every language."""
   constant_defs = read_json_file(filename);
   constants_text = '\n'
   for key in constant_defs:
     value = constant_defs[key]
     value = value.replace('"', '\\"')
-    constants_text += '\nBlockly.Msg.{0} = \"{1}\";'.format(key, value)  
```

Comment:
Messages are now being referred to by name: "%{BKY_BLAH_BLAH_BLAH}"  As a result, they must be exported as named properties.